src/main/java/cz/fi/muni/pa165/esports/entity/MatchRecord.java

```diff
@@ -64,12 +59,12 @@ public void setScore(int score) {
         this.score = score;
     }
 
-    public long getMatch_number() {
-        return match_number;
+    public long getMatchnumber() {
+        return matchnumber;
```

Comment:
Also please I see you renamed `match_number` to `matchnumber`, but it should be `matchNumber` to be consistent with the class diagram :)